fontbe/src/avar.rs

```diff
@@ -21,7 +22,30 @@ pub fn create_avar_work() -> Box<BeWork> {
     Box::new(AvarWork {})
 }
 
-fn to_segment_map(axis: &Axis) -> Option<SegmentMaps> {
+/// Return true if the SegmentMaps is not a boring identity map
+fn is_interesting(segment_map: &SegmentMaps) -> bool {
+    segment_map
+        .axis_value_maps
+        .iter()
+        .any(|av| av.from_coordinate != av.to_coordinate)
+}
+
+/// Return a default avar SegmentMaps containing the required {-1:-1, 0:0, 1:1} maps
+fn default_segment_map() -> SegmentMaps {
```

Comment:
and this feels like it should probably be the impl of `Default` for `SegmentMaps`; however we currently autogenerate `Default` impls for all write types, and there's no mechanism to override this, so this is probably fine.